api.bs

```diff
@@ -395,6 +395,39 @@ the {{PrivateAttributionConversionOptions/aggregator}} member of the
 {{PrivateAttributionConversionOptions}} dictionary when calling the
 <a method for=PrivateAttribution>measureConversion()</a> method.
 
+## Site Identities ## {#sites}
+
+This API relies on the HTML definition of [=site=]
+as the primary scope over which it operates.
+Three types of sites are recognized:
+
+*   An <dfn>impression site</dfn> is the [=site=]
+    derived from the [=top-level origin=]
+    of the [=relevant settings object=]
+    at the time that {{PrivateAttribution/saveImpression()}}
+    is invoked to store an [=impression=].
+
+*   A <dfn>conversion site</dfn> is the [=site=]
+    derived from the [=top-level origin=]
+    of the [=relevant settings object=]
+    at the time that {{PrivateAttribution/measureConversion()}} is invoked.
+
+*   A <dfn>intermediary site</dfn> is the [=site=]
+    derived from the [=origin=]
+    of the [=relevant settings object=]
+    at the time that either {{PrivateAttribution/saveImpression()}}
+    or {{PrivateAttribution/measureConversion()}} is invoked,
+    unless this origin is the same as the [=impression site=]
```

Comment:
Should we link `same` to some definition of `same site` / `same origin`, or is this referring to some other concept?

Reply:
```suggestion
    unless this origin is [=same site=] with the [=impression site=]
```